web/packages/shared/setupTests.tsx

```diff
@@ -29,3 +43,24 @@ global.ResizeObserver = jest.fn().mockImplementation(() => ({
   unobserve: jest.fn(),
   disconnect: jest.fn(),
 }));
+
+const rootDir = path.join(__dirname, '..', '..', '..');
+// Do not add new paths to this list, instead fix the underlying problem which causes console.error
+// or console.warn to be used.
+//
+// If the test is expected to use either of those console functions, follow the advice from the
+// error message.
+const failOnConsoleIgnoreList = new Set([
+  'web/packages/design/src/utils/match/matchers.test.ts',
+  'web/packages/shared/components/TextEditor/TextEditor.test.tsx',
+  'web/packages/teleport/src/components/BannerList/useAlerts.test.tsx',
+  'web/packages/teleport/src/Navigation/NavigationItem.test.tsx',
+  'web/packages/teleterm/src/ui/TabHost/TabHost.test.tsx',
```

Comment:
i was testing it out and we may need to add one more?
![Uploading image.png…]()

Reply:
It just failed on CI though. ;f I'm going to add it to the list then.

https://github.com/gravitational/teleport/actions/runs/9015736034/job/24770973952#step:7:1125

Reply:
Same thing with `web/packages/teleport/src/Recordings/Recordings.story.test.tsx` locally just now. Definitely a flake since I cannot even reproduce it with the same seed.